AdNaviConstructor:
  server_type:
    service: "service"
    action: "action"

FastLoose:
  acc_lim_x: 1.0 
  acc_lim_y: 0.0
  acc_lim_theta: 0.5
  acc_lim_theta: 0.7
  max_vel_trans: 0.3
  min_vel_trans: 0.0
  max_vel_x: 0.6
  min_vel_x: 0.0
  max_vel_y: 0.0
  min_vel_y: 0.0
  max_vel_theta: 0.7
  min_vel_theta: 0.0

  yaw_goal_tolerance: 0.15
  xy_goal_tolerance: 0.50
  trans_stopped_vel: 0.1
  theta_stopped_vel: 0.1

Default:
  restore_defaults: True #(bool default: false)

